deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -358,9 +368,15 @@ class ChatViewController: UITableViewController {
                         self.scrollToLastUnseenMessage()
                     }
                 }, completion: { [weak self] finished in
+                    guard let self = self else { return }
                     if finished {
-                        self?.isInitial = false
-                        self?.ignoreInputBarChange = false
+                        if !self.isInitial {
+                            // relaod messages since this VC was in background in the navigation stack,
+                            // all message observers have been inactive in the meanwhile
+                            self.loadMessages()
```

Comment:
in general, i am wondering why we cannot leave leave observers active somehow and do this `loadMessages()` only when there are really changes, maybe just by a single flag.

if feels so much faster when this call is skipped!

maybe this is sth. for another pr, however, the loadMessages() call was there before as well.